src/exos-scripts.ts

```diff
@@ -1,10 +1,11 @@
 import chalk from "chalk";
 import path from "path";
 import spawn from "cross-spawn";
+import type { ExosScriptsNames } from "./common/types";
 
-const availableScripts: string[] = ["build", "start", "lint", "test", "stylelint"];
+const availableScripts: ExosScriptsNames[] = ["build", "start", "lint", "test", "stylelint"];
```

Comment:
I see this list repeating in various places. Why not to create enum for it?

Reply:
Just to make it simpler, but yes, having it twice is annoying.

Reply:
Done